backend/tests/unit/repositories/test_history_repository.py

```
@@ -1033,6 +1036,69 @@ def test_mixed_successful_and_failed_interactions_in_session(self, repository, s
+        repository.session.commit()
+        repository.session.refresh(stage_execution)
+        
+        # Now update it to completed
+        stage_execution.status = StageStatus.COMPLETED
+        stage_execution.completed_at_us = now_us()
+        stage_execution.duration_ms = 5000
+        stage_execution.stage_output = {"result": "analysis complete"}
+        
+        # Update the stage execution
+        result = repository.update_stage_execution(stage_execution)
+        assert result == True
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `result:` for truth checks

Replace with `result`

(E712)